### DISCUSSION TITLE
Listbox option value

### DISCUSSION THREAD

**Discussion by WiSiHe** [tailwindlabs/headlessui#657] [Help]
Heya, I was just wondering about the Option for the listbox, 

`<Listbox.Options static>
              {people.map((person) => (
                <Listbox.Option key={person.id} value={person}>
                  {person.name}
                </Listbox.Option>
              ))}
            </Listbox.Options>`

is there any reason as to why the listbox.option is using the object as a value, and not {person.name}? :)

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

Ideally you want a **unique** value so that the identity of a certain object is clear. Objects in our examples are unique and can be used as a value.
That said, you can also use primitive values like an `id` that is a number or a string.

So `person.name` can be used, but I recommend you **not** to do that, because you need to guarantee uniqueness, names are often not unique. In production code I would use an object or an `id`.
